src/main/java/org/folio/circulation/domain/policy/lostitem/ProcessingChargeFee.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.circulation.domain.policy.lostitem;
+
+import java.math.BigDecimal;
+
+public final class ProcessingChargeFee extends ChargeFee {
+  public ProcessingChargeFee(BigDecimal amount) {
+    super(amount);
+  }
+}
```

Comment:
Why does a processing fee need a specific type? There appears to be no difference between it and another kind of fee?

Reply:
The idea was to distinguish set cost and actual cost fees. But now when actual cost is postponed, we can use a single class for both fees.

Reply:
These specific classes were removed.